Tests/ProjectDescriptionTests/TargetTests.swift

```diff
@@ -32,7 +32,7 @@ final class TargetTests: XCTestCase {
                              coreDataModels: [CoreDataModel("pat", currentVersion: "version")],
                              environment: ["a": "b"])
 
-        let expected = "{\"headers\":{\"public\":\"public\\/*\",\"private\":\"private\\/*\",\"project\":\"project\\/*\"},\"bundle_id\":\"bundle_id\",\"core_data_models\":[{\"path\":\"pat\",\"current_version\":\"version\"}],\"actions\":[{\"arguments\":[\"arg\"],\"path\":\"path\",\"order\":\"post\",\"name\":\"name\"}],\"product\":\"app\",\"sources\":{\"globs\":[\"sources\\/*\"]},\"settings\":{\"base\":{\"a\":\"b\"},\"debug\":{\"xcconfig\":\"config\",\"settings\":{\"a\":\"b\"}},\"release\":{\"xcconfig\":\"config\",\"settings\":{\"a\":\"b\"}}},\"resources\":[{\"type\":\"glob\",\"pattern\":\"resources\\/*\"}],\"platform\":\"ios\",\"entitlements\":\"entitlement\",\"info_plist\":{\"type\": \"file\",\"path\":\"info.plist\"},\"dependencies\":[{\"type\":\"framework\",\"path\":\"path\"},{\"path\":\"path\",\"public_headers\":\"public\",\"swift_module_map\":\"module\",\"type\":\"library\"},{\"type\":\"project\",\"target\":\"target\",\"path\":\"path\"},{\"type\":\"target\",\"name\":\"name\"}],\"environment\":{\"a\":\"b\"},\"name\":\"name\"}"
+        let expected = "{\"headers\":{\"public\":\"public\\/*\",\"private\":\"private\\/*\",\"project\":\"project\\/*\"},\"bundle_id\":\"bundle_id\",\"core_data_models\":[{\"path\":\"pat\",\"current_version\":\"version\"}],\"actions\":[{\"arguments\":[\"arg\"],\"path\":\"path\",\"order\":\"post\",\"name\":\"name\"}],\"product\":\"app\",\"sources\":{\"globs\":[\"sources\\/*\"]},\"settings\":{\"base\":{\"a\":\"b\"},\"debug\":{\"xcconfig\":\"config\",\"settings\":{\"a\":\"b\"}},\"release\":{\"xcconfig\":\"config\",\"settings\":{\"a\":\"b\"}},\"defaultSettings\":\"recommended\"},\"resources\":[{\"type\":\"glob\",\"pattern\":\"resources\\/*\"}],\"platform\":\"ios\",\"entitlements\":\"entitlement\",\"info_plist\":{\"type\": \"file\",\"path\":\"info.plist\"},\"dependencies\":[{\"type\":\"framework\",\"path\":\"path\"},{\"path\":\"path\",\"public_headers\":\"public\",\"swift_module_map\":\"module\",\"type\":\"library\"},{\"type\":\"project\",\"target\":\"target\",\"path\":\"path\"},{\"type\":\"target\",\"name\":\"name\"}],\"environment\":{\"a\":\"b\"},\"name\":\"name\"}"
```

Comment:
Did you decide if you wanted to format these JSON structures?

Reply:
Yes, I do, I will format it once https://github.com/tuist/tuist/pull/377 is landed.